daprdocs/content/en/reference/annotations/annotations-overview.md

```diff
@@ -0,0 +1,55 @@
+---
+type: docs
+title: "Dapr annotations spec"
+linkTitle: "Daprd, CLI, and Kubernetes annotations"
+description: "The annotations available when configuring Dapr in different environments"
+weight: 300
+---
+
+The following table shows all the annotations supported by Dapr:
```

Comment:
I would add a few more lines of explanation here stating that this table is meant to help users understand the equivalent options for running Dapr sidecars in different contexts - via the CLI directly via daprd or on Kubernetes via annotations. For each, adding a link to the relevant doc that explains more (CLI reference, daprd doc and Kubernetes overview)